### PR TITLE
Finished Login/Sign-Up Authentication

### PR DESCRIPTION
**Pull request by CalebObsCode**
Added Routing
Added Database
Implemented GET and POST to fetch data from SQlite

### CODE REVIEW

File: Installation.md [hunk 1]
```diff
@@ -3,4 +3,6 @@ All installation module should be stored here:
 
 ## Set Server to Debug to auto-update localhost
 FLASK_DEBUG=1 flask run
```
Comment (CalebObsCode) [inline Installation.md:5]: Still in the process of setting up the installation readme so everyone can install flask and other dependencies to run the web app locally.

File: WayneSociety_Project/app.py [hunk 2]
```diff
@@ -1,10 +1,16 @@
 from flask import Blueprint
```
Comment (CalebObsCode) [inline WayneSociety_Project/app.py:1]: Moving this objective to Sprint 2.

**Comment by AnushRajam** [comment]
Good Job Caleb!

**Comment by Ashraf7199** [comment]
Great job with connecting the database Caleb!

**Pull request merged**